### DISCUSSION TITLE
Custom config file path and name

### DISCUSSION THREAD

**Discussion by gaevoc** [thinkst/opencanary#282] [Feature Requests]
Hello,
first of all thank you for your great work. 
My question is about if there any way to specify a custom config file location and name for opencanaryd at startup. I would like to use a path which is not in the predefined sequence.

Thank you
Gaetano

**Comment by jayjb** [comment]
Hi @gaevoc,

Currently, we don't support a custom config file location. It should be something easy enough to add though and we do accept PRs ;) I'd be happy to help you get it sorted
